src/vehicle_state/vehicle_classes.py

```diff
@@ -130,29 +131,37 @@ def _add_node_to_vehicle(self, node_purpose):
         node_type = NodeType(node_purpose['type'])
         node_name = node_purpose['name']
 
+        # ID used for HTTPSERVER this is not similar as
+        # the node ids for the canbus network.
+        id_index = 0
```

Comment:
misschien kan je het beter zo noemen dan?
```suggestion
        http_index = 0
```